docs/source/dynamo/troubleshooting.rst

```diff
@@ -597,7 +597,7 @@ recompile that function (or part) up to
 hitting the cache limit, you will first need to determine which guard is
 failing and what part of your program is triggering it.
 
-The `recompilation profiler <#recompilation-profiler>`__ automates the
+The `compile profiler <https://github.com/pytorch/pytorch/blob/b26efd0dd27949b9b5087f646e1834fb43c91aa9/torch/_dynamo/utils.py#L925>`__ automates the
```

Comment:
prefer non-permalink just path on master?

Reply:
Well, the problem with non-permalink is it can get silently broken. What will be better is if we generate documentation for torch._dynamo (or make some public hierarchy at torch.dynamo) and link to the docs